packages/atomic/src/components/insight/atomic-insight-generated-answer/atomic-insight-generated-answer.tsx

```diff
@@ -229,15 +247,32 @@ export class AtomicInsightGeneratedAnswer
     );
   }
 
+  private validateMaxCollapsedHeight(): number {
+    const isValid =
+      this.maxCollapsedHeight >= this.MIN_COLLAPSED_HEIGHT &&
+      this.maxCollapsedHeight <= this.MAX_COLLAPSED_HEIGHT;
+
+    return isValid ? this.maxCollapsedHeight : this.DEFAULT_COLLAPSED_HEIGHT;
```

Comment:
I think when it's invalid we should output a warning because it's indicative of a misconfiguration.